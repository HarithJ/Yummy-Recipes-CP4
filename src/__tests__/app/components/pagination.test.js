import React from 'react';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson, { shallowToJson } from 'enzyme-to-json';

import Pagination from '../../../components/pagination.js'

describe("pagination component", () => {
  const pagination = shallow(<Pagination pagination={{totalPages: 3}}/>)

  it("renders properly", () => {
    expect(shallowToJson(pagination)).toMatchSnapshot();
  })
})
