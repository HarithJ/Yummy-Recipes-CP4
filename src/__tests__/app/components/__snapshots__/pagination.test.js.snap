// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pagination component renders properly 1`] = `
<div>
  <nav
    aria-label="Page navigation example"
    className="fixed-bottom mr-1  justify-content-end"
  >
    <ul
      className="pagination justify-content-end"
    >
      <li
        className="page-item undefined"
      >
        <input
          className="page-link"
          onClick={[Function]}
          tabIndex="-1"
          type="button"
          value="Previous"
        />
      </li>
      <li
        className="page-item"
        key="0"
      >
        <input
          className="page-link"
          onClick={[Function]}
          type="button"
          value={1}
        />
      </li>
      <li
        className="page-item"
        key="1"
      >
        <input
          className="page-link"
          onClick={[Function]}
          type="button"
          value={2}
        />
      </li>
      <li
        className="page-item"
        key="2"
      >
        <input
          className="page-link"
          onClick={[Function]}
          type="button"
          value={3}
        />
      </li>
      <li
        className="page-item undefined"
      >
        <input
          className="page-link"
          onClick={[Function]}
          type="button"
          value="Next"
        />
      </li>
      <select
        onChange={[Function]}
      >
        <option
          value="0"
        >
          All
        </option>
        <option
          value="1"
        >
          1
        </option>
        <option
          value="10"
        >
          10
        </option>
        <option
          value="20"
        >
          20
        </option>
        <option
          value="30"
        >
          30
        </option>
      </select>
    </ul>
  </nav>
</div>
`;
